e2e/tests/ui-driven/src/create-flow-with-geospatial.spec.ts

```diff
@@ -99,12 +118,41 @@ test.describe("Flow creation, publish and preview", () => {
     await page.goto(
       `/${context.team.slug}/${serviceProps.slug}/published?analytics=false`,
     );
+
     await expect(
       page.locator("h1", { hasText: "Find the property" }),
     ).toBeVisible();
     await answerFindProperty(page);
     await clickContinue({ page });
 
+    await expect(
+      page.getByRole("heading", { name: "About the property" }),
+    ).toBeVisible();
+
+    // Check map component has geoJson content
+    await checkGeoJsonContent(page, mockMapGeoJson);
+
+    // Check property info is being shown
+    await expect(page.getByText("Test Street, Testville")).toBeVisible();
+    await expect(page.getByText("Residential - Semi Detached")).toBeVisible();
+    const changeButton = page.getByRole("button", {
+      name: "Change your Property type",
+    });
+
+    await changeButton.click();
+
+    // change button navigates the user back thro the service so we need to continue twice
```

Comment:
When I initially land on this page, it'd be useful to check that the "Residential" option is selected by default based on the mocked property type value of "Residential - Semi Detached" above _before_ changing the answer to "Commercial". 

A key functionality/expectation of all PlanX "back" navigation is that previous answers are pre-populated/retained but we're failing to check that in this test.